docs/CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 * Searching mods and perks in D2 now searches non-selected perks as well.
 * Perks are in the correct order again (instead of the selected one being first always).
 * Unpurchaseable vendor items are displayed better.
+* Ratings and reviews for vendors and collections.
```

Comment:
We shouldn't put this in the changelog yet, since it isn't going to release. Vendors/collections is still beta-only.

Reply:
(thumbs up emoji)